lib/options/uefi-setup-menu.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Documentation       Library for UEFI configuration using the UEFI setup menu
+...                 app (e.g. over serial port)
+
+Library             Collections
+Library             String
+Resource            ../bios/menus.robot
+
+
+*** Keywords ***
+Set UEFI Option
+    [Documentation]    Set an UEFI option to a value.
+    ...    TODO: Only works with options following the submenu/submenu/option
```

Comment:
Do we have an idea which limitations it brings, and how to overcome this? I would prefer if we create and link issues also in the code right away, if we intend to merge a TODO comment.

Reply:
We can probably handle this by checking the length of the path. I'll try to handle this

Reply:
handled in 2b89ab672a6bcf42a4226bb58e67079cdb1bc985